spec/resources/users_spec.rb

```diff
@@ -6,12 +6,12 @@
   describe '#create' do
     # http://onlinehelp.tableau.com/v9.0/api/rest_api/en-us/help.htm#REST/rest_api_ref.htm#Add_User_to_Site%3FTocPath%3DAPI%2520Reference%7C_____7
     it 'can create a user in a site' do
-      user = client.users.create(username: 'test', site_role: 'Unlicensed')
+      user = client.users.create(username: 'test', site_role: 'ExplorerCanPublish')
```

Comment:
is this a tableau API change?

Reply:
Yeah, the roles are the one breaking change they said.